docs/README.md

```diff
@@ -38,6 +38,7 @@ The `trustpilot` app also enables Truspilot's widget component to be added to yo
 6. Save your changes;
```

Comment:
A good practice when writing steps is to finish each one with a period or a colon as we can check [here](https://www.notion.so/vtexhandbook/Editorial-guidelines-33cb21cab65844d5b35a7a2e1bf87056#b1748ecd144d42619754c2f7492f3261), for example: 
```suggestion
6. Save your changes.
```

Reply:
fixed